mm2src/coins/nft.rs

```diff
@@ -44,9 +44,15 @@ pub async fn get_nft_list(ctx: MmArc, req: NftListReq) -> MmResult<NftList, GetN
             NftListStorageOps::init(&storage, chain).await?;
         }
     }
-    let nfts = storage
+    let mut nfts = storage
         .get_nft_list(req.chains, req.max, req.limit, req.page_number)
         .await?;
+    if req.protect_from_spam {
+        for nft in &mut nfts.nfts {
```

Comment:
`nfts.nfts` seems little weird, maybe find better name for the variable?

Reply:
agree, changed it to `nft_list`, as from storage we get `NftList` structure